packages/yourloops/test/components/dashboard-widgets/medical-files/weekly-reports-dialog.test.tsx

```diff
@@ -0,0 +1,85 @@
+/**
+ * Copyright (c) 2022, Diabeloop
+ *
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import React from "react";
+import { fireEvent, render, screen } from "@testing-library/react";
+import WeeklyReportDialog from "../../../../components/dialogs/weekly-report-dialog";
+import { WeeklyReport } from "../../../../lib/medical-files/model";
+import * as teamHookMock from "../../../../lib/team";
+import { createPatient, createPatientTeam } from "../../../common/utils";
+import { UserInvitationStatus } from "../../../../models/generic";
+import { UNITS_TYPE } from "../../../../lib/units/utils";
+import { Alarm } from "../../../../models/alarm";
+
+
+jest.mock("../../../../lib/team");
+describe("Weekly report dialog", () => {
+  const teamId = "teamId";
+  const patientId = "patientId";
+  const patient = createPatient(patientId, [createPatientTeam(teamId, UserInvitationStatus.accepted)]);
+  const onClose = jest.fn();
+  const weeklyReport: WeeklyReport = {
+    id: "fakeId",
+    patientId,
+    teamId,
+    parameters: {
+      bgUnit: UNITS_TYPE.MGDL,
+      lowBg: 1,
+      highBg: 2,
+      outOfRangeThreshold: 3,
+      veryLowBg: 4,
+      hypoThreshold: 5,
+      nonDataTxThreshold: 6,
+      reportingPeriod: 7,
+    },
+    alarms: {} as Alarm,
+    creationDate: "2022-02-02",
+  };
+
+  function renderComponent() {
+    render(<WeeklyReportDialog onClose={onClose} weeklyReport={weeklyReport} />);
+  }
+
+
+  beforeAll(() => {
+    (teamHookMock.useTeam as jest.Mock).mockImplementation(() => {
+      return { getPatient: () => patient };
+    });
+  });
+
+  it("should render dialog", () => {
```

Comment:
I don't think that this tests is really useful (as long as there are other tests on this components)